lib/pyper/pipes/cassandra/paging_reader.rb

```diff
@@ -0,0 +1,40 @@
+module Pyper::Pipes::Cassandra
+  # A pipe for reading all items from a single row in cassandra
+  # @param [Symbol] table name
+  # @param [Cassava::Client] client to query cassandra with
+  # @param [Hash] Additional/default options to pass to the Cassava execute statement.
+  class PagingReader < Struct.new(:table, :client, :options)
```

Comment:
the options param doesn't seem to be used anywhere -- it's not actually passed to execute.